.goreleaser.yml

```diff
@@ -0,0 +1,29 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
```

Comment:
Since goreleaser config file is commited, can you suggest a way to install it?
Maybe a *goreleaser* target in `Makefile`?

Reply:
I add goleaser task in Makefile, but not sure how to make it much useful, as the default install dir is ./bin/goreleaser

Reply:
Fair enough! ;-)